### DISCUSSION TITLE
ERROR: openssl/1.1.1i: Error in build() method, line 762 	self._make() while calling '_make', line 684

### DISCUSSION THREAD

**Discussion by daisyvincent** [openssl/openssl#21537] [Q&A]
I am try to build a source code in   ubuntu20.04 with
conan 1.57
cmake 3.13.2
gcc: 9.4.0
gstreamer

 i get the below error

openssl/1.1.1i: ERROR: Package '8faa1e71bc57b8654248632a449b4c6957f7dcd5' build failed
openssl/1.1.1i: WARN: Build folder /home//AlexaSDK_4_0/alexa-auto-sdk-4.0/builder/.builder/.conan/data/openssl/1.1.1i/_/_/build/8faa1e71bc57b8654248632a449b4c6957f7dcd5
ERROR: openssl/1.1.1i: Error in build() method, line 762
	self._make()
while calling '_make', line 684
	self.run('{perl} ./Configure {args}'.format(perl=self._perl, args=args), win_bash=self._win_bash)
	ConanException: Error 1 while executing perl ./Configure "conan-Release-Android-armv8-clang-8" no-shared --prefix=

can u tell the solution for solving the error

**Comment by t8m** [comment]
I am sorry but that does not look like a bug reported from our build system.

**Comment by mattcaswell** [comment]
Looks a conan issue. Probably you should report this problem there.
